src/library/returnBook/ReturnBookControl.java

```diff
@@ -0,0 +1,95 @@
+package library.returnBook;
+import library.entities.Item;
+import library.entities.Library;
+import library.entities.Loan;
+
+public class ReturnBookControl {
+
+    private ReturnBookUI ui;
+
+    private enum ControlState { INITIALISED, READY, INSPECTING };
+
+    private ControlState state;
+    
+    private Library library;
+
+    private Loan currentLoan;
+    
+
+    public ReturnBookControl() {
```

Comment:
#PR41 - D3: 
Constructor isn't listed in class diagram.. though please rename to ReturnItemControl.

Reply:
https://github.com/elliotmitchum/ITC205/pull/41/commits/1774fdc9d206d8411b12829d60f152f49b04b6dc